Shipr-Containerization/shipr-frontend/nginx.conf

```diff
@@ -0,0 +1,16 @@
+server {
+    listen 80;
+    server_name localhost;
+
+    root /usr/share/nginx/html;
+    index index.html;
+
+    location / {
+        try_files $uri /index.html;
+    }
+
+    error_page 404 /index.html;
+    location = /404.html {
+        internal;
+    }
+}
```

Comment:
This could be improved greatly. Also, please have a look at the [documentation](https://hub.docker.com/_/nginx) for official nginx image, you may want to redefine following directives.

![image](https://github.com/user-attachments/assets/0c12da2a-8098-498b-aa5b-76df376bec62)

Reply:
Only writing `USER nginx` in Dockerfile is not safe you also need to make the changes which are in the screenshot of documentation. Alternatively you can use this base image https://hub.docker.com/r/nginxinc/nginx-unprivileged

Reply:
Used nginxinc/nginx-unprivileged image in dockerfile